packages/bls12-381/bls12-381.0.3.15/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "OCaml binding for bls12-381 from librustzcash"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "conf-rust" {build}
+  "ocaml" {>= "4.07"}
+  "dune" {>= "1.7"}
```

Comment:
Your `dune-project` file states that this package requires at least dune 2.7, not 1.7

Reply:
The dune-project states it requires 2.0 (this release comes from the branch `match-tezos`, built over `0.3.14` as the master branch contains already some changes not integrated yet in `tezos-rust-libs`). Changing.
This development process will change in the next weeks when `tezos-rust-libs` will integrate the changes required for `bls12-381.0.4.0`.